app/controllers/concerns/program_permissions.rb

```diff
@@ -31,7 +31,7 @@ module ProgramPermissions
     end
 
     private def check_edit_permission!
-      not_authorized! unless can_edit_programs? || (can_edit_assigned_programs? && @subprogram.editable_by?(current_user))
+      not_authorized! unless can_edit_programs? || (can_edit_assigned_programs? && @subprogram&.editable_by?(current_user))
```

Comment:
Seems off that `@subprogram` would ever be nil, but it fails safe...

Reply:
Yeah, we've been seeing odd can't do fu on nil, so I just spammed them around a bit.